src/asInput/index.jsx

```diff
@@ -27,7 +27,7 @@ export const inputProps = {
   onBlur: PropTypes.func,
   validator: PropTypes.func,
   isValid: PropTypes.bool,
-  validationMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  invalidMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
Alright, I'll also be that person - I'm not sure `invalidMessage` is the most appropriate name here. 

When I see `invalidMessage` I think of a message that's not valid or a `boolean` that represents if a message is valid or not, not necessarily a message that will be displayed if a value is invalid.

Maybe `invalidValueMessage` is slightly more accurate?

Could also be the case that

![alt-text](https://media.giphy.com/media/3ornjVl19PVacFaRa0/giphy.gif)

Reply:
@jaebradley What do you think? On the one hand it's kind of the crux of the thing, but on the other it's not a big deal to change. Happy to go either way.

Reply:
Err I meant to make this comment about [line 67](https://github.com/edx/paragon/pull/240/files#diff-8f7eccce481920c8f772c0263869dde4R67), the internal variable called `invalidMessage` (and not the prop).

It might be worth discussing with @arizzitano, @thallada, and @edx/fedx-team ?

Reply:
@jaebradley You raise a valid point, but I think this is ok. Shipping as is.